### PR TITLE
Don't fail ncp-backup-auto

### PR DESCRIPTION
**Pull request by llucax**
Make sure the script don't return an exit status different from 0 so the
cron job doesn't send e-mails on failure, as notifications are being
delivered as Nextcloud notifications. Also redirect all output to a log
file, so in case of failure it is possible to review what happened.

This also makes the notification a bit more informative, as it says
which script failed.

The log files is overwritten on each run. This should not be a problem
since backups are run more frequently than once per day, so there should
be a fair amount of time to see the backup logs.

### CODE REVIEW

**Comment by nachoparker** [comment]
thanks for the PR, overall looks good, let me know when it's finished

**Comment by llucax** [comment]
Hi, you are welcome, thanks for putting nexcloudpi together :)

From my side I'd say it is finished, but let me know if I missing anything.

**Comment by nachoparker** [comment]
can we output the log to `ncp.log`? the reason is that we have proper log rotation set up for that one

File: bin/ncp/BACKUPS/nc-backup-auto.sh [hunk 1]
```diff
@@ -19,17 +19,28 @@ configure()
...
+run_script after
+if [[ -n "\$failed" ]]
+then
+  notify_admin "Auto-backup failed" "The \$failed backup script(s) failed"
+fi
 EOF
   chmod +x /usr/local/bin/ncp-backup-auto
 
-  echo "0  3  */${BACKUPDAYS}  *  *  root  /usr/local/bin/ncp-backup-auto" > /etc/cron.d/ncp-backup-auto
+  echo "0  3  */${BACKUPDAYS}  *  *  root  /usr/local/bin/ncp-backup-auto > /var/log/ncp-backup-auto.log 2>&1" > /etc/cron.d/ncp-backup-auto
```
Comment (nachoparker) [inline bin/ncp/BACKUPS/nc-backup-auto.sh:43]:
```suggestion
  echo "0  3  */${BACKUPDAYS}  *  *  root  /usr/local/bin/ncp-backup-auto > /var/log/ncp.log 2>&1" > /etc/cron.d/ncp-backup-auto
```

**Comment by nachoparker** [comment]
thanks!

**Pull request merged**